stdlib/json.rb

```diff
@@ -54,7 +54,7 @@ class ParserError < JSONError
             }
 
             if (!options.parse && (klass = #{`hash`[JSON.create_id]}) != nil) {
-              klass = Opal.get(klass);
+              klass = Opal.const_get_qualified('::', klass);
```

Comment:
@iliabylich I think the important point here is that we should define the public runtime API before 1.0